packages/bot-web-ui/src/components/dashboard/dashboard-components/index.scss

```diff
@@ -0,0 +1,166 @@
+.bot-dashboard {
+    .toolbar__section {
+        justify-content: end;
+    }
+    .run-panel__container {
+        height: calc(100vh - 15rem) !important;
+        position: unset !important;
+        transform: unset !important;
+        margin-right: -350px;
+        transition: all 0.4s;
+        margin-top: 48px;
+    }
+    .dc-drawer--open {
+        margin-right: 0 !important;
+    }
+}
+.dashboard {
+    &__container {
+        border: none;
+        max-width: 100%;
+        // border: solid 1px var(--general-section-1);
+        background: #fff;
+        margin-right: 1rem;
+        width: 70%;
+
+        .dc-tabs {
+            &__active-line {
+                background: none;
+            }
+
+            &__active {
+                background: #fff;
+            }
+
+            &__list {
+                background: #e5e5e5;
+                justify-content: start;
+            }
+
+            &__item {
+                border-radius: 1rem 1rem 0px 0px;
+                height: 48px;
+                width: 14.4rem;
+                display: flex;
+                justify-content: center;
+                align-items: center;
+                svg {
+                    width: 1.6rem;
+                    height: 1.6rem;
+                    padding-right: 0;
+                    margin-right: 0.8rem;
+                }
+            }
+
+            &__content_group {
+                display: flex;
+                flex-direction: column;
+                justify-content: center;
+                align-items: center;
+                height: calc(100vh - 20rem);
+                margin-top: 1rem;
+
+                &_heading {
+                    font-size: 20px;
```

Comment:
BEM css naming does not contain `-` ... if it's a block it should be  specified by `.`  if it is an element it should be `__` and if it is a modifier it should be `--` ... this is the acceptable convention ... 

Refer to:
http://getbem.com/naming/